releasenotes-builder/src/main/java/com/github/checkstyle/MainProcess.java

```diff
@@ -79,7 +80,81 @@ public static List<String> runPostGenerationAndPublication(
             Multimap<String, ReleaseNotesMessage> releaseNotes, CliOptions cliOptions)
             throws IOException, TemplateException {
         runPostGeneration(releaseNotes, cliOptions);
-        return runPostPublication(cliOptions);
+        final List<String> errors = new ArrayList<>();
+        if (cliOptions.isValidateVersion() != null) {
```

Comment:
I am having trouble finding the place to set the default value of `validateVersion` to `false`. I see that some values do get a default value, others `null`.

Reply:
Nevermind, found it.